Client/Frontend/Browser/Onboarding/onboarding-ads.json

```diff
@@ -1 +1 @@
":{"a":0,"k":{"i":[[0,0],[0,0],[0,0],[0.46,-0.46],[0.3,0],[0.22,0.23],[0,0],[0,0],[0.3,0],[0.23,0.23],[-0.45,0.45],[0,0],[0,0],[-0.46,0.46],[-0.45,-0.46],[0,0],[0,0],[-0.46,-0.46],[0.45,-0.45]],"o":[[0,0],[0,0],[0.46,0.45],[-0.23,0.23],[-0.3,0],[0,0],[0,0],[-0.22,0.22],[-0.3,0],[-0.46,-0.46],[0,0],[0,0],[-0.46,-0.45],[0.46,-0.46],[0,0],[0,0],[0.45,-0.45],[0.45,0.46],[0,0]],"v":[[4.345,-2.695],[1.655,-0.005],[4.325,2.675],[4.325,4.325],[3.495,4.665],[2.675,4.325],[-0.005,1.655],[-2.695,4.345],[-3.515,4.685],[-4.345,4.345],[-4.345,2.695],[-1.655,0.005],[-4.325,-2.675],[-4.325,-4.325],[-2.675,-4.325],[0.005,-1.655],[2.695,-4.345],[4.345,-4.345],[4.345,-2.695]],"c":true},"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[1,1,1,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[300,300],"ix":3},"r":{"a":0,"k":-315,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Path Copy 6","np":2,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":1830.20007454557,"st":0,"bm":0},{"ddd":0,"ind":18,"ty":4,"nm":"float-text1","sr":1,"ks":{"o":{"a":1,"k":[{"i":{"x":[0.833],"y":[0.833]},"o":{"x":[0.167],"y":[0.167]},"t":22,"s":[0]},{"t":25.0000010182709,"s":[100]}],"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":1,"k":[{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":25,"s":[218.469,475.856,0],"to":[-10.467,-21.233,0],"ti":[-1.022,71.768,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":45,"s":[155.667,348.456,0],"to":[1.022,-71.768,0],"ti":[8.825,124.262,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":65,"s":[224.601,45.246,0],"to":[-8.825,-124.262,0],"ti":[20.314,73.727,0]},{"t":85.000003462121,"s":[102.715,-397.117,0]}],"ix":2},"a":{"a":0,"k":[0,0,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":0,"ef":[{"ty":25,"nm":"Drop Shadow","np":8,"mn":"ADBE Drop Shadow","ix":1,"en":1,"ef":[{"ty":2,"nm":"Shadow Color","mn":"ADBE Drop Shadow-0001","ix":1,"v":{"a":0,"k":[0.152941003442,0.180391997099,0.250979989767,0.20000000298],"ix":1}},{"ty":0,"nm":"Opacity","mn":"ADBE Drop Shadow-0002","ix":2,"v":{"a":0,"k":51,"ix":2}},{"ty":0,"nm":"Direction","mn":"ADBE Drop Shadow-0003","ix":3,"v":{"a":0,"k":180,"ix":3}},{"ty":0,"nm":"Distance","mn":"ADBE Drop Shadow-0004","ix":4,"v":{"a":0,"k":15,"ix":4}},{"ty":0,"nm":"Softness","mn":"ADBE Drop Shadow-0005","ix":5,"v":{"a":0,"k":33,"ix":5}},{"ty":7,"nm":"Shadow Only","mn":"ADBE Drop Shadow-0006","ix":6,"v":{"a":0,"k":0,"ix":6}}]}],"shapes":[{"ty":"gr","it":[{"d":1,"ty":"el","s":{"a":0,"k":[69,67.444],"ix":2},"p":{"a":0,"k":[0,0],"ix":3},"nm":"Ellipse Path 1","mn":"ADBE Vector Shape - Ellipse","hd":false},{"ty":"fl","c":{"a":0,"k":[0.517647027969,0.368627011776,0.968626976013,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn
```

Comment:
?

Reply:
This is part of the ticket. https://github.com/brave/brave-rewards-ios/issues/239
I was messaged to add it in last night on Slack so I added it but only pushed it this morning.
Basically the animation changed for both the REWARDS and ADS screen.